src/EPROMStore.cpp

```diff
@@ -383,8 +385,16 @@ float EEPROMStore::getRAStepsPerDegree()
 {
     float raStepsPerDegree(RA_STEPS_PER_DEGREE);  // Default value
 
-    if (isPresent(RA_STEPS_FLAG))
+    // Latest version stores 100x steps/deg for 256 MS
+    if (isPresentExtended(RA_NORM_STEPS_MARKER_FLAG))
+    {
+        float factor     = 25600.0 / RA_TRACKING_MICROSTEPPING;
```

Comment:
Refactor magic number `25600.0` into something meaningful

Reply:
Any suggestions? Scale256x100? NormedTo256MSx100? The comment above the line explains what it is. StorageScaling?

Reply:
So we're assuming 256 is the maximum supported microstep setting? (I know it's the max on TMCs)
Maybe have a constant in `EPROMStore.hpp` `maxSupportedMicrostepping = 256`, then something like `storageScaling = 100.0`?

Reply:
It could support higher microstepping. I could also normalize it to 1.0 (instead of 25600) and store a float.